.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,59 @@
+# this is reference workflow to run e2e tests and get test results
+name: e2e-tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - playwright_initial_HDS-2224
+
```

Comment:
I guess this has to be changed before merge?

Reply:
Yes this will be changed. 

e2e-tests.yml is a reference workflow and most probably will be removed.  e2e tests should be run on hds-demo-preview -workflow (HDS-1589)